shared/settings/account/index.tsx

```diff
@@ -11,6 +11,8 @@ export type Props = {
   addedEmail: string | null
   contactKeys: I.List<string>
   hasPassword: boolean
+  supersededPhoneNumber?: ContactRowProps
```

Comment:
This doesn't really need to be a full `ContactRowProps`, does it? Can you use only the one or two fiels of that you actually need?